src/System.Text.Json/src/System/Text/Json/Reader/JsonReaderHelper.Unescaping.cs

```diff
@@ -107,6 +132,46 @@ public static bool UnescapeAndCompare(ReadOnlySequence<byte> utf8Source, ReadOnl
             return result;
         }
 
+        public static bool TryDecoderInPlaceHelper(Span<byte> utf8Unescaped, out byte[] bytes)
+        {
+            OperationStatus status = Base64.DecodeFromUtf8InPlace(utf8Unescaped, out int bytesWritten);
+            if (status != OperationStatus.Done)
+            {
+                bytes = null;
+                return false;
+            }
+            bytes = utf8Unescaped.Slice(0, bytesWritten).ToArray();
+            return true;
+        }
+
+        public static bool TryDecoderHelper(ReadOnlySpan<byte> utf8Unescaped, out byte[] bytes)
```

Comment:
Why is `Helper` in the method name? It's already in the type name, and it's not being used to have a public-verifying and a private-trusting way of doing the same work.